src/UniswapV2ERC20.sol

```diff
@@ -1,100 +0,0 @@
-/* solhint-disable const-name-snakecase */
-pragma solidity ^0.8.0;
-
-import { IUniswapV2ERC20 } from "src/interfaces/IUniswapV2ERC20.sol";
-
-// TODO: Is UniswapV2ERC20 still the best ERC-20 to inherit from or should we
```

Comment:
indeed, solmate's one is more gas efficient and funnily is based on uni's impl. 

I've replaced it with such